src/App.js

```diff
@@ -0,0 +1,32 @@
+import './App.css';
+import Navbar from './components/Navbar/Navbar';
+import Button from './Asset/Button/Button';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+const App = () => {
+  const [setUser] = useState({});
+  const [temp, settemp] = useState([]);
+  const getUser = async () => {
+    let randomNumber = Math.floor(Math.random() * 10);
+    const res = await axios.get(
+      `https://api.unsplash.com/search/photos?query=london&client_id=${process.env.REACT_APP_CLIENT_ID}`
```

Comment:
create one configuration/config folder at src level and create SERVER_CONFIG.js file into it and keep all static url's withing it